controllers/clustertopology_controller_sync.go

```diff
@@ -16,10 +16,69 @@ limitations under the License.
 
```

Comment:
Can we move the patch helper to a _mergepatch file?

Reply:
Done!